src/libmongoc/src/mongoc/mongoc-rpc.c

```diff
@@ -993,6 +997,29 @@ _mongoc_rpc_get_first_document (mongoc_rpc_t *rpc, bson_t *reply)
    return false;
 }
 
+/* Reply to a first OP_MSG with another OP_MSG: */
```

Comment:
The comment suggests that this function does the reply. I suggest clarifying this parses a BSON document from a reply. 

```suggestion
/* Get the first BSON document from an OP_MSG reply: */
```